### DISCUSSION TITLE
Jerarquía de Estilos en CSS

### DISCUSSION THREAD

**Discussion by Ivano93** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#170] [Show and tell]
¡Buena noche a todos, explorers!
Para quienes tenían curiosidad acerca de cómo se van sobrescribiendo los estilos al declararlos en CSS, van así (mayor a menor importancia):

1. !important tag
`.first-class { color: pink !important; }`
2. inline styles (declaración del estilo directo en el bloque de HTML)
`<h1 style="color: red;"> Hello World! </h1>`
3. Id tags
`#id-tag{ color: orange; }`
4. clases (clases subsecuentes sobrescriben clases previas)
`.first-class { color: pink; }`
`.second-class { color: blue; }`
5. Estilos generales de bloque (tipo body, h1, etc.)
`body { color: green; }`

Por ejemplo, si tuviéramos todos estos estilos juntos:
![image](https://user-images.githubusercontent.com/71421542/156096869-db2530ed-19dd-49a4-8897-39243b5315af.png)

La salida del texto sería en color ROSA, por la declaración del !important tag en la clase ".first-class":
![image](https://user-images.githubusercontent.com/71421542/156095810-d6c02191-e1ba-46d6-a45e-57fb309dd1c5.png)

Espero les sea de ayuda.
Saludos,

H. Iván Hdz.

> Referencias: Larson, Q. (s.f.). freeCodeCamp. Obtenido de Responsive Web Design, Basic CSS: 
> https://www.freecodecamp.org/learn/responsive-web-design/#basic-css

**Comment by JuanSalvador73** [comment]
Gracias por la info.
